tests/test_search.py

```diff
@@ -176,7 +179,10 @@ def test_IndexBuilder():
         'index': {'docname1_1', 'docname1_2', 'docname2_1', 'docname2_2'},
         'test': {'docname1_1', 'docname1_2', 'docname2_1', 'docname2_2'},
     }
-    assert index._title_mapping == {'section_titl': {'docname1_1', 'docname1_2', 'docname2_1', 'docname2_2'}}
+    assert index._title_mapping == {
+        'another_titl': {'docname1_1', 'docname1_2', 'docname2_1', 'docname2_2'},
```

Comment:
Can you use a shorter search term? without the "e", I almost thought it was a typo and wanted to correct it. For instance, how about `another_ti`? (same question for `section_*` and other cases)

Reply:
In this case, `another_titl` isn't a user-entered query, but is the [stemmed](https://en.wikipedia.org/wiki/Stemming) form of the title that would be stored in the index file (`searchindex.js`).

We could try to modify the test documents so that they use heading names that look like valid English even after stemming - what do you think?

Reply:
Oh, no, let's not overcomplicate the things. I think a document telling us (people who don't work on that component) how the test API works would be good actually, like a README, or something like that, but it should be another PR.